Gruntfile.js

```diff
@@ -427,7 +442,7 @@ module.exports = (grunt) => {
   // Run tests without compile (use watcher or manually build)
   grunt.registerTask('unittest', ['mochaTest:unit']);
   grunt.registerTask('clicktest', ['mochaTest:click']);
-  grunt.registerTask('test', ['unittest', 'clicktest']);
+  grunt.registerTask('test', ['checkPrettier', 'unittest', 'clicktest']);
```

Comment:
Since this is now in the `default` task I think this can be remove from here. I don't like being interruppted by code style issue when modifying / debugging tests :)